Index: src/main/java/model/database/DatabaseManager.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/model/database/DatabaseManager.java b/src/main/java/database/DatabaseManager.java
rename from src/main/java/model/database/DatabaseManager.java
rename to src/main/java/database/DatabaseManager.java
--- a/src/main/java/model/database/DatabaseManager.java	(revision 68fdb2ce28208fa979c71e969c21b16096423904)
+++ b/src/main/java/database/DatabaseManager.java	(date 1780595447380)
@@ -1,4 +1,4 @@
-package model.database;
+package database;
 
 import com.google.gson.Gson;
 import com.google.gson.reflect.TypeToken;
Index: src/main/java/model/database/DatabaseRecord.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/model/database/DatabaseRecord.java b/src/main/java/database/DatabaseRecord.java
rename from src/main/java/model/database/DatabaseRecord.java
rename to src/main/java/database/DatabaseRecord.java
--- a/src/main/java/model/database/DatabaseRecord.java	(revision 68fdb2ce28208fa979c71e969c21b16096423904)
+++ b/src/main/java/database/DatabaseRecord.java	(date 1780595447389)
@@ -1,4 +1,4 @@
-package model.database;
+package database;
 
 public class DatabaseRecord {
     
Index: src/main/java/model/database/DatabaseConnection.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/model/database/DatabaseConnection.java b/src/main/java/database/DatabaseConnection.java
rename from src/main/java/model/database/DatabaseConnection.java
rename to src/main/java/database/DatabaseConnection.java
--- a/src/main/java/model/database/DatabaseConnection.java	(revision 68fdb2ce28208fa979c71e969c21b16096423904)
+++ b/src/main/java/database/DatabaseConnection.java	(date 1780595447368)
@@ -1,6 +1,4 @@
-package model.database;
-
-import model.database.DatabaseRecord;
+package database;
 
 import java.util.Optional;
 public interface DatabaseConnection {
Index: src/main/java/model/database/EncryptionEngine.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/model/database/EncryptionEngine.java b/src/main/java/database/EncryptionEngine.java
rename from src/main/java/model/database/EncryptionEngine.java
rename to src/main/java/database/EncryptionEngine.java
--- a/src/main/java/model/database/EncryptionEngine.java	(revision 68fdb2ce28208fa979c71e969c21b16096423904)
+++ b/src/main/java/database/EncryptionEngine.java	(date 1780595447397)
@@ -1,4 +1,4 @@
-package model.database;
+package database;
 
 
 public class EncryptionEngine {
